src/common/main.lua

```diff
@@ -324,6 +324,49 @@ function ant.init()
 			Names = msg.Tags.Names,
 		})
 	end)
+
+	--[[
+	AOS provides a _boot handler that is designed to load Lua code on boot.
+	This handler OVERRIDES this and replaces it with our ANT state initialization handler.
+
+	]]
+	Handlers.once("_boot", function(msg)
+		return msg.Tags.Type == "Process" and Owner == msg.From
+	end, function(msg)
+		if type(msg.Data) == "string" then
+			-- If data is present assume its an attempt to initialize the state
+			local status, res = xpcall(function()
+				initialize.initializeANTState(msg.Data)
```

Comment:
nit: add documentation to initialize state describing the function and call signature of function

Reply:
[7ae6ea9](https://github.com/ar-io/ar-io-ant-process/pull/57/commits/7ae6ea96be130fa58eb20eef79bc252e5aa289f3)

Updated the readme as well.